src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -17,7 +17,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       Defines the list of file extensions that VS will monitor to reload the application.
       We'll only define these for C# projects targeting RazorLangVersion 3.0 or later, and let VS use defaults in other cases.
     -->
-    <UpToDateReloadFileTypes Condition="'$(Language)'=='C#' AND '$(_Targeting30OrNewerRazorLangVersion)' == 'true'">$(UpToDateReloadFileTypes);.cs;.razor;.resx;.cshtml</UpToDateReloadFileTypes>
+    <UpToDateReloadFileTypes Condition="'$(Language)'=='C#' AND '$(_Targeting30OrNewerRazorLangVersion)' == 'true' AND '$(RazorUpToDateReloadFileTypes)' == ''">$(UpToDateReloadFileTypes);$(RazorUpToDateReloadFileTypes)</UpToDateReloadFileTypes>
```

Comment:
Here's the follow up PR: https://github.com/aspnet/AspNetCore/pull/10716

The target in the package is evaluated before this target. Consequently it does not have a way to affect it. Moreoever, this always prepends the existing value, consequently even if we produce the right result when RuntimeCompilation is involved, we add .cshtml files back in to the mix.

This introduces a new property that packages may modify and assigns that to `UpToDateReloadFileTypes` if the right conditions are met. I copied this property to the `Sdk.Razor.CurrentVersion.targets` to unblock us until we do VS insertion.